data/map.txt

```diff
@@ -27384,6 +27532,15 @@ planet "Graveyard Wormhole (Unstable)"
 planet "Graveyard Wormhole (Stable)"
 	description ``
 
+planet Greymoon
+	attributes station hai
+	landscape land/space6
+	description `This station is owned and operated by the Far Loo Bah fuel collection corporation. This is a fact that you cannot avoid seeing, as their logo is emblazoned on nearly every possible surface and sign indicating their sponsorship of every service or feature on the station. Interestingly this includes a "History of Spaceflight" museum which takes up an entire branch of the station. Naturally, the museum focuses very heavily on the Far Loo Bah corporation's role in that history.`
```

Comment:
```suggestion
	description `This station is owned and operated by the Far Loo Bah fuel collection corporation. This is a fact that you cannot avoid seeing, as their logo is emblazoned on nearly every possible surface and sign. Interestingly this includes a "History of Spaceflight" museum which takes up an entire branch of the station. Naturally, the museum focuses very heavily on the Far Loo Bah corporation's role in that history.`
```

Reply:
The next sentence now doesn't quite follow, as the museum is presented as one of the features they've sponsored.

Reply:
All right, fair enough.